internal/storage/block_storage_test.go

```diff
@@ -343,7 +350,11 @@ func TestBalance(t *testing.T) {
 			Value:    "-1000",
 			Currency: currency,
 		}
-		mockHelper = &MockBlockStorageHelper{}
+		mockHelper = &MockBlockStorageHelper{
```

Comment:
You shouldn't need to make any modifications to the `MockBlockStorageHelper` to implement this test.

Because the account key doesn't exist, we shouldn't hit any calls to the helper.

Reply:
You are right after the new switch case, this won't be hit. However, this will be help catch regressions, because without the fix, the helper will be called and it needs to return a positive balance for genesis accounts. Without this it would return zero, hiding the test failure.

Reply:
🤦 ahhhh yeah I gotcha...great call.

You could accomplish the same thing by using the already existing `AccountBalanceAmount` field (however, I see how adding the ability to stub specific account balances will be useful):
```suggestion
		mockHelper = &MockBlockStorageHelper{
		  AccountBalanceAmount: "100",
		}
```

I thought that deprecating `AccountBalanceAmount` may be a good idea but it is still useful (to stub any account amount).

Reply:
I just included this suggestion to show other ways to do this ^^ but your change allows for more powerful testing later(preferred).